components/asp/SystemProcessor.py

```
@@ -2,6 +2,10 @@
 from Cdl import Cdl
 import json
 from pathlib import Path
+from database.TableWriter import TableWriter
+
+from TraceAssembler import TraceAssembler
```

Ruff (0.8.2)

`TraceAssembler.TraceAssembler` imported but unused

Remove unused import: `TraceAssembler.TraceAssembler`

(F401)